app/fixture.json

```diff
@@ -709,6 +709,16 @@
         "updated_at": "2021-03-06T20:21:13.811Z",
         "image": "https://i.postimg.cc/nr9xt4hh/Skjermbilde-2021-01-31-kl-19-31-24.png",
         "image_alt": "TIHLDE",
+        "creator": {
+            "image": "https://i.pinimg.com/custom_covers/222x/85498161615209203_1636332751.jpg",
+            "image_alt": "Profilbilde",
+            "first_name": "Martin",
+            "last_name": "Fredrik",
+            "email": "Fredrik@martin.no",
+            "gender":2,
+            "user_class":3,
+            "user_study": 1
+        },
```

Comment:
Har du skrevet dette selv eller kjørt `make dumpdata`? Når det er fremmednøkler så skal det i utgangspunktet stå id til det relaterte objektet i fixtures (se registration).

Reply:
Skrevet selv🤪 kjører `make dumpdata` neste gang! Takk for kommando